src/CppAst/CppModelBuilder.cs

```diff
@@ -435,12 +493,38 @@ private CXChildVisitResult VisitMember(CXCursor cursor, CXCursor parent, void* d
                 case CXCursorKind.CXCursor_CXXBaseSpecifier:
                     {
                         var cppClass = (CppClass)GetOrCreateDeclarationContainer(parent, data).Container;
-                        var baseType = GetCppType(cursor.Type.Declaration, cursor.Type, cursor, data);
+                        var baseType = GetCppType(cursor.Type.Declaration, cursor.Type, cursor, data);
+
+                        /** This is kind of a hack to get the specialized template arguments */
+                        CXCursor paramCursor;
+                        CXType paramCursorType;
+                        if (cursor.Type.kind == CXTypeKind.CXType_Elaborated)
+                        {
+                            paramCursor = cursor.Type.CanonicalType.Declaration;
+                            paramCursorType = cursor.Type.CanonicalType;
+                        }
+                        else
+                        {
+                            paramCursor = cursor.Type.Declaration;
+                            paramCursorType = cursor.Type;
+                        }
+                        var templateParameters = ParseTemplateSpecializedArguments(paramCursor, paramCursorType, new CXClientData((IntPtr)data));
```

Comment:
It seems like you only need the specialized arguments. can you take only the arguments instead of the tuple?
```suggestion
                        var (_, templateSpecializedArguments) = ParseTemplateSpecializedArguments(paramCursor, paramCursorType, new CXClientData((IntPtr)data));
```